.github/workflows/release-swift-toolchain-binary-sizes.yml

```diff
@@ -153,10 +153,11 @@ jobs:
 
       - uses: actions/setup-python@v5
         with:
-          python-version: '3.9'
+          python-version: '3.9.10'
           cache: 'pip' 
+          architecture: x64
```

Comment:
That works properly on arm64? I assume this ends up using the x64 Python binaries.

Reply:
yes. it uses the emulator, which works for the bolaty binaries. the other option is to install arm64 version ourselves from nuget and set it in the path etc..

Reply:
> That works properly on arm64

this job has both amd and arm: https://github.com/thebrowsercompany/swift-build/actions/runs/16761621354/job/47458026308